docs/sources/mimir/release-notes/v2.13.md

```diff
@@ -0,0 +1,85 @@
+<!-- vale Grafana.Timeless = NO -->
+<!-- Release notes are often future focused -->
+
+Grafana Labs is excited to announce version 2.13 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bug fixes in this release.
+For the complete list of changes, refer to the [CHANGELOG](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Improved CPU performance in processing queries with regular expressions** that contain many alternations (for example `foo|bar|baz|...`).
+
+- **Improved OTLP ingestion performance** by using native translation, which reduces memory usage in the distributor by up to 30% and CPU usage by up to 8%.
+
+- Configurable S3 bucket lookup type improves interoperability with **S3-compatible providers such as Tencent and Alibaba**. Configure lookup type via `-<prefix>.s3.bucket-lookup-type` or `-common.storage.s3-bucket-lookup-type`.
+
+- **Configuration of TLS for S3 buckets** is now possible via `-<prefix>.s3.http.*` and `-common.storage.s3.http.*` configuration options.
+
+- **Mimirtool can now verify the validity of a Mimir runtime configuration** file with the `mimirtool runtime-config verify` command.
+
+- **Active series are now updated along with owned series.** This means the number of active series for a tenant is much more accurate after scaling out ingesters. As a result ingesters now enforce more precisely tenants' series limits. This is only enabled when `-ingester.track-ingester-owned-series` or `-ingester.use-ingester-owned-series-for-limits` are enabled.
+
+- **Store-gateway can be configured to explicitly disable or enable tenants**. This is useful in cases when compaction slows down for a tenant to temporarily exclude a tenant while compaction catches up without impact other tenants.
+
+- **Remote read (`/prometheus/api/v1/read`) is becoming a first-class endpoint in Mimir** with support in `query stats` logs and query blocking in the query-frontend. More coming in future releases.
+
+Additionally, the following previously experimental features are now considered stable:
+
+- Rules tenant federation via the `source_tenants` field in rule groups.
+- Enabling and recording and alerting rules evaluation on a per-tenant basis.
+- Limiting the number of tenants a federated query can query.
+
+## Important changes
+
+In Grafana Mimir 2.13 the following behavior has changed:
+
+- The default Docker image `grafana/mimir` is now based on the distroless image `gcr.io/distroless/static-debian12`.
+  See [Debugging distroless container images](https://grafana.com/docs/mimir/next/manage/mimir-runbooks/#debugging-distroless-container-images-in-kubernetes) for more details on how to
+  work with distroless images.
+
+- Error logs in the ingester are now sampled at 10% by default. Sampled log lines will contain `(sampled 1/10)`. All the discarded samples will still be tracked by the `cortex_discarded_samples_total` metric.
```

Comment:
```suggestion
- Error logs in the ingester are now sampled at 10% by default. Sampled log lines contain `(sampled 1/10)`. The `cortex_discarded_samples_total` metric still tracks all discarded samples.
```

Reply:
Small update to avoid "will" and to use active voice, per https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/.